rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "nightly-2023-03-28"
+components = [ "rustfmt", "rustc-dev", "clippy", "llvm-tools" ]
+targets = [ "x86_64-unknown-linux-gnu" ]
+profile = "default"
```

Comment:
as default, maybe we don't need `rustc-dev`. and most developers use macOS so the targets also don't need, and the profile maybe also can remove.

We just need the version and the default components for compile the app.